www/artists/wkurelek/gallery.html

```diff
@@ -35,7 +35,7 @@ <h1>
                     <br>
                     <img src="who_has_seen_the_wind.jpg" alt="Who Has Seen The Wind" width="300" height="300" align="left">
                     <br>
-                    William did the illustration for Who Has Seen the Wind in 1998.
+                    William did the illustration for Who Has Seen the Wind in 1974.
```

Comment:
May I request a source on the new year? I don't want innaccurate information on the site.

Reply:
This is the source that Wikipedia cited
https://web.archive.org/web/20090202164827/http://www.saskpublishers.sk.ca/sampler/spotlight/wo3.htm